bin/docpress

```diff
@@ -29,6 +35,23 @@ function run () {
   }
 }
 
+function help () {
```

Comment:
Agree, feel free to edit/violate this PR :)  
![](https://link.nylas.com/open/45iu4p1xszs6wds5wnj30hzg1/local-8bb6ed2d-96c9?
r=cmVwbHkrMDAwMTIyOTE4MjMyMDc4NmQ3NzA3MDI3YjYzNDdmNDViZjJkMzQ4YTgzYWYwM2FmOTJj
ZjAwMDAwMDAxMTNkYzVjNmE5MmExNzBjZTA0OTIwYzdlQHJlcGx5LmdpdGh1Yi5jb20=)

  
On Aug 30 2016, at 4:26 am, Kiko Beats <notifications@github.com> wrote:  

> In [bin/docpress](https://github.com/docpress/docpress/pull/147#discussion_r
76680318&r=cmVwbHkrMDAwMTIyOTE4MjMyMDc4NmQ3NzA3MDI3YjYzNDdmNDViZjJkMzQ4YTgzYWY
wM2FmOTJjZjAwMDAwMDAxMTNkYzVjNmE5MmExNzBjZTA0OTIwYzdlQHJlcGx5LmdpdGh1Yi5jb20=)
:

>  
>  
>     > @@ -29,6 +35,23 @@ function run () {

>     >    }

>     >  }

>     >  
>     > +function help () {

>  
>

> A more clean would be:

>

>   * put the content of the help command in a file, for example: [https://git
hub.com/bumped/bumped/blob/master/bin/help.txt](https://github.com/bumped/bump
ed/blob/master/bin/help.txt&r=cmVwbHkrMDAwMTIyOTE4MjMyMDc4NmQ3NzA3MDI3YjYzNDdm
NDViZjJkMzQ4YTgzYWYwM2FmOTJjZjAwMDAwMDAxMTNkYzVjNmE5MmExNzBjZTA0OTIwYzdlQHJlcG
x5LmdpdGh1Yi5jb20=)

>   * load the file! [https://github.com/bumped/bumped/blob/master/bin/index.j
s#L10](https://github.com/bumped/bumped/blob/master/bin/index.js#L10&r=cmVwbHk
rMDAwMTIyOTE4MjMyMDc4NmQ3NzA3MDI3YjYzNDdmNDViZjJkMzQ4YTgzYWYwM2FmOTJjZjAwMDAwM
DAxMTNkYzVjNmE5MmExNzBjZTA0OTIwYzdlQHJlcGx5LmdpdGh1Yi5jb20=)

>

> --  
You are receiving this because you authored the thread.  
Reply to this email directly, [view it on GitHub](https://github.com/docpress/
docpress/pull/147/files/0673f2a35be09b0b8d0d3c7fd283d4e5a8a802e4#r76680318&r=c
mVwbHkrMDAwMTIyOTE4MjMyMDc4NmQ3NzA3MDI3YjYzNDdmNDViZjJkMzQ4YTgzYWYwM2FmOTJjZjA
wMDAwMDAxMTNkYzVjNmE5MmExNzBjZTA0OTIwYzdlQHJlcGx5LmdpdGh1Yi5jb20=), or [mute
the thread](https://github.com/notifications/unsubscribe-auth/AAEikYJ_sYZp0VKw
BsFv4L32Q_UINeChks5qk0BqgaJpZM4Jv1N9&r=cmVwbHkrMDAwMTIyOTE4MjMyMDc4NmQ3NzA3MDI
3YjYzNDdmNDViZjJkMzQ4YTgzYWYwM2FmOTJjZjAwMDAwMDAxMTNkYzVjNmE5MmExNzBjZTA0OTIwY
zdlQHJlcGx5LmdpdGh1Yi5jb20=).![](https://github.com/notifications/beacon
/AAEikSw8Bn-v608h3IjoxiM5jOlEsFYvks5qk0BqgaJpZM4Jv1N9.gif)